package.json

```diff
@@ -52,14 +61,19 @@
     "eslint": "^8.45.0",
     "eslint-config-next": "^14.0.0",
     "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-storybook": "^0.6.15",
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "image-size": "^1.0.2",
     "mdast-util-toc": "^7.0.0",
     "minimist": "^1.2.8",
     "polished": "^4.2.2",
+    "storybook": "7.5.3",
     "ts-node": "^10.9.1",
     "typescript": "^5.1.6",
     "unified": "^10.0.0",
     "unist-util-visit": "^5.0.0"
+  },
+  "resolutions": {
+    "jackspeak": "2.1.1"
```

Comment:
Is there any special reason why we need to use this specific version? Noticed that this is a dep of `storybook`

Reply:
There is an issue with another dependency in storybook tied to the use of jackspeak and working Yarn v1

https://github.com/storybookjs/storybook/issues/22431#issuecomment-1630086092